Update change logs to match released versions (#40)
smileidentity/smile-identity-core-ruby: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
## [2.0.0] - 2022-11-08
- build: Adds support for ruby 3.1
- build: Drops support for ruby 2.5.1
- ci: Move from TravisCI to Github Actions
- core: Enforces the use of signature on all API calls
- core: Removes support for `sec_key` as an authentication method
- core: Adds helper constants SMILE_IDENTITY_CORE::ENV, SMILE_IDENTITY_CORE::IMAGE_TYPE, SMILE_IDENTITY_CORE::JOB_TYPE
- docs: adds "examples" folder
  

> ## [1.2.1]

Reviewer: ```suggestion
## [1.2.1] - 2021-12-02
```

- Revert changes to version SmileIdentityCore.version_as_hash

> ## [1.2.0] 

Reviewer: ```suggestion
## [1.2.0] - 2021-10-14
```

- Change the urls for both test and prod aa55a72d10854f05a35db4dad3ea63930e8996f6
- Add Signature option for signing requests

> ## [1.1.0]

Reviewer: ```suggestion
## [1.1.0] - 2021-09-30
```

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

> ## [2.0.0] - 2022-04-28
> - Migrate from travis to github actions.
> - Format codebase using rubocop.
> - Add support for ruby 3.1
> - Drop support for ruby 2.5.1
> - Add "examples" folder
> - Add helper constants SMILE_IDENTITY_CORE::ENV, SMILE_IDENTITY_CORE::IMAGE_TYPE, SMILE_IDENTITY_CORE::JOB_TYPE
> - Added source and version number to all requests
> - Fixed verifications with sec_key failing
>  Enforced the use of signature on all API calls
> - Removed the use of sec key on all API calls

Reviewer 2: makes sense to have this as our version 2.
I would re-order these though.

Language run-time updates as top

so this would look like this

```suggestion
## [2.0.0] - 2022-11-08
- build: Adds support for ruby 3.1
- build: Drops support for ruby 2.5.1
- ci: Move from TravisCI to Github Actions
- core: Enforces the use of signature on all API calls
- core: Removes support for `sec_key` as an authentication method
- core: Adds helper constants SMILE_IDENTITY_CORE::ENV, SMILE_IDENTITY_CORE::IMAGE_TYPE, SMILE_IDENTITY_CORE::JOB_TYPE
- docs: adds "examples" folder
```
